databricks_cli/sdk/service.py

```diff
@@ -706,3 +706,45 @@ def get_groups_for_principal(self, user_name=None, group_name=None, headers=None
             _data['group_name'] = group_name
         return self.client.perform_query('GET', '/groups/list-parents', data=_data, headers=headers)
 
+
+class DeltaPipelinesService(object):
+    def __init__(self, client):
+        self.client = client
+
+    def deploy(self, pipeline_id=None, id=None, name=None, storage=None, filters=None,
+               clusters=None, libraries=None, transformations=None, credentials=None,
+               headers=None):
+        _data = {}
+        if pipeline_id is not None:
+            _data['pipeline_id'] = pipeline_id
+        if id is not None:
+            _data['id'] = id
+        if name is not None:
+            _data['name'] = name
+        if storage is not None:
+            _data['storage'] = storage
+        if filters is not None:
+            _data['filters'] = filters
+            if not isinstance(filters, dict):
+                raise TypeError('Expected databricks.Filters() or dict for field filters')
```

Comment:
This is from the autogeneration, I'm not completely sure why it only checks Filters and Credentials. 
Also this is being bypassed because of the self.client.client